docs/docs-content/automation/palette-cli/commands/ec.md

```diff
@@ -37,15 +42,16 @@ you for required values. Alternatively, you can use flags to generate a configur
-| `-v`       | `--custom-values-file` | Enterprise Cluster custom values.yaml configuration file (optional). Use this to customize the cluster profile of the Enterprise Cluster. Refer to the [custom value file](#custom-value-file) section for more information. | string  |
-| `-p`       | `--update-passwords`   | Update passwords only. Do not proceed with installation. The `--config-file` flag must also be provided.                                                                                                                     | string  |
-| `-t`       | `--update-tokens`      | Update authentication tokens only. Do not proceed with installation. The `--config-file` flag must be provided.                                                                                                              | boolean |
-| -          | `--validate`           | Scan the environment and conduct validation before the enterprise cluster is installed.                                                                                                                                      | boolean |
+| Short Flag | Long Flag                 | Description                                                                                                                                                                                                                                                                                                                                                                 | Type    |
+| ---------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `-f`       | `--config-file`           | Install using a configuration file (optional). Use `-config-only` to generate a configuration file.                                                                                                                                                                                                                                                                         | string  |
+| `-d`       | `--skip-teardown`         | Skip tear down of the kind cluster in case of errors.                                                                                                                                                                                                                                                                                                                       | boolean |
```

Comment:
```suggestion
| `-d`       | `--skip-teardown`         | Skip teardown of the kind cluster in case of errors.                                                                                                                                                                                                                                                                                                                       | boolean |
```